pkg/aetosutil/dashboardutil.go

```diff
@@ -513,3 +521,26 @@ func Get() *Dashboard {
 	}
 	return dash
 }
+
+func (d *Dashboard) UpdateStats(name, product, statType, version string, dashStats map[string]string) {
+	if d.IsEnabled {
+
+		dashStats["dash-url"] = fmt.Sprintf("http://aetos.pwx.purestorage.com/resultSet/testSetID/%d", d.TestSetID)
```

Comment:
There is a constant defined for AETOS URL, can you please use that here and in other places

Reply:
fixed